src/eddymotion/model.py

```diff
@@ -264,18 +264,19 @@ def __init__(self, gtab, **kwargs):
             Whether the summary statistic to apply is ``"mean"`` or ``"median"``.
 
         """
-        self._gtab = gtab
         self._th_low = kwargs.get("th_low", 50)
-        self._th_high = kwargs.get("th_high", self._gtab[3, ...].max())
+        self._th_high = kwargs.get("th_high", 10000)
```

Comment:
I know this is just a toy model, but we might want to document the default values for these, in case they get inherited into other places.